File: PyWebView/native_extension/DotNet_binding/pythonnet_classlib/HelloLibrary/HelloService.cs
namespace HelloLibrary;

public sealed class HelloService
{
    private readonly string _name;

    public HelloService(string name)
    {
        _name = name;
    }

    public string Greet()
    {
        return $"hello, {_name} from .NET!";
    }

    public static int Add(int left, int right)
    {
        return left + right;
    }
}

File: PyWebView/native_extension/Java_binding/jpype_jar/src/main/java/com/example/HelloService.java
package com.example;

public class HelloService {
    private final String name;

    public HelloService(String name) {
        this.name = name;
    }

    public String greet() {
        return "hello, " + name + " from Java!";
    }

    public static int add(int left, int right) {
        return left + right;
    }
}
